.travis.yml

```diff
@@ -13,17 +13,15 @@ before_cache:
 
 matrix:
   include:
-    # crashes in travis because 2 versions of unix get installed and
```

Comment:
can we keep this block for further reference?

Reply:
Sure

Reply:
Thanks, seems like this line is still missing.

Reply:
Fixed that too.